js/stores/windowStore.js

```diff
@@ -206,8 +209,12 @@ const doAction = (action) => {
       break
     case WindowConstants.WINDOW_NEW_FRAME:
       let nextKey = incrementNextKey()
+      let nextPartitionNumber = currentPartitionNumber
+      if (action.frameOpts.isPartitioned) {
+        nextPartitionNumber = incrementPartitionNumber()
+      }
```

Comment:
This seems buggy. if i open a new tab (via cmd+t), it seems to always uses the highest partition number. Ex: after closing all my partition-2 tabs, I open a tab from my default partition tab and it is in partition 2.

I expected new tabs to inherit the partition of the opener tab.